src/main/java/mocacong/server/controller/MemberController.java

```diff
@@ -129,6 +126,17 @@ public ResponseEntity<Void> updateProfileInfo(
         return ResponseEntity.ok().build();
     }
 
+    @Operation(summary = "비밀번호 확인 인증")
+    @SecurityRequirement(name = "JWT")
+    @PostMapping("/info/password")
+    public ResponseEntity<PasswordVerifyResponse> passwordVerify(
+            @LoginUserEmail String email,
+            @RequestBody @Valid PasswordVerifyRequest request
+    ) {
+        PasswordVerifyResponse response = memberService.verifyPassword(email, request.getPassword());
```

Comment:
민감정보 로그 마스킹 전략 상 해당 로직을 아래처럼 수정해야될 거 같아요
```suggestion
        PasswordVerifyResponse response = memberService.verifyPassword(email, request);
```

request를 넘기는 경우
`password=abcabc` 형태로 넘어오고 Logger가 password 뒤 부분을 마스킹해줌.

request.getPassword()를 넘기는 경우
`abcabc` 형태로 넘어오고 Logger는 비밀번호인지 알 턱이 없기 때문에 마스킹을 안해줌.

Reply:
그 부분을 놓쳤네요. 수정하겠습니다 👍